_minutes/README.md

```diff
@@ -0,0 +1,23 @@
+# Meetings
+
+The [WebExtensions Community group](https://www.w3.org/community/webextensions/) meets virtually every other week, for one hour.
+Instructions to join the meeting are sent to the group's mailing list at https://lists.w3.org/Archives/Member/internal-webextensions/
+
+To accomodate attendees across the globe, the recurring meeting alternates between two time slots:
+
+* Thursday 8 AM PDT / Thursday 3 PM UTC
+* Thursday 11 PM PDT / Friday 6 AM UTC
+* To convert to your local time zone, see https://everytimezone.com/
+
+After the end of each meeting, meeting notes are published here.
```

Comment:
Should we include anything here about the process? For example, "the meeting's scribe will submit a PR with the minutes. Attendees are encouraged to review and request edits as appropriate. The PR will be merged after N days."

Reply:
Probably not needed. We haven't really established a process. And in the past two meetings, nobody asked for revisions of the minutes on Github.

Reply:
Fair point :)